project_euler/problem_187/sol1.py

```diff
@@ -68,6 +70,32 @@ def calculate_prime_numbers(max_number: int) -> list[int]:
     return [2] + [2 * i + 1 for i in range(1, max_number // 2) if is_prime[i]]
 
 
+def np_calculate_prime_numbers(max_number: int) -> list[int]:
```

Comment:
Is it possible to make this as a helper function in a common module as I see it being used in your other PR as well or maybe add it in `maths/` and re-use it here or (assuming that there's already similar function in `maths/`) re-use an existing implementation?

Reply:
Hi @dhruvmanila,
Please see if I added the helper function correctly.

Also I found out that the benchmark in the file [maths/prime_numbers.py](https://github.com/TheAlgorithms/Python/tree/2e28bde50d33b1e655e457eaf46196f185b5c4c5/maths/prime_numbers.py) was not working as expected, so I fixed it.